content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting.rst

```diff
@@ -1,214 +1,99 @@
-be paid to the subcontractor for his parts and service time: **B +
-s**. The product cost has to be: **A + B + s**, how much the
-product is valued in the accounting.
-
-Finally, the subcontractor bill then matches the purchase order, with
-the proposed price coming from the finished products C.
-
-.. note::
-      If managing the replenishment of raw materials **B** at your
-      subcontractor’s location is not needed, simply include the cost of
-      **B** in the subcontractor’s price **s** and remove the products
-      *B* from the *BoM*.
-
-Traceability
-============
-
-In case the received products from the subcontractor contain tracked
-components, their serial or lot numbers need to be specified during the
-receipt.
-
-In that case, on the receipt of the subcontracted product, a *Record Components*
-button appears. Click on it to open a dialog box and record the serial/lot numbers of
-the components. If the finished product is also tracked, its serial/lot number can
-be registered here too.
-
-.. image:: subcontracting/sbc_3.png
-    :align: center
-    :alt:
-
-For audit purposes, it is possible to check the lot numbers recorded on
-a receipt by using the icon on the right of the finished products:
-
-.. image:: subcontracting/sbc_4.png
-    :align: center
-    :alt:
-
-Also note that in case flexible consumption has been selected on the subcontracted
-BOM for a non-tracked product, the record components option will also appear
-optionally on each move line, if you want to register more or less component consumption
-at your subcontracting location, when receiving your final product.
-
-.. image:: subcontracting/sbc_5.png
-    :align: center
-    :alt:
-
-As you can see, the reception of both of these non-tracked products can either be executed
-by selecting the 'Set Quantities' Option or via the move line hamburger menus.
-
-Automate Replenishment of Subcontractors
-========================================
-
-There are two ways to automate the supply of raw materials to your
-subcontractors when purchasing the final product. The chosen method
-depends on whether or not you want the materials to transit through your
-warehouse. Both of these methods are described as pull style mechanisms as
-their trigger is the inital PO to the subcontractor, which creates a need
-at the subcontracting location, for raw material.
-
-If you are supplying your subcontractor with raw material from your own warehouse,
-you must activate the 'Resupply Subcontractor on Order' route as shown
-below. If this is a component that you buy from a vendor, the buy route
-should also be activated.
-
-
-.. image:: subcontracting/sbc_6.png
-    :align: center
-    :alt:
+Configuration
+=============
 
+To enable subcontracting in Odoo, navigate to :menuselection:`Manufacturing app --> Configuration
+--> Settings`, and enable the checkbox next to :guilabel:`Subcontracting`, under the
+:guilabel:`Operations` heading.
```

Comment:
I personally like to include an image of the setting enabled, to 
1) break up the text with an image
2) readers typically like images